content/project-organization/quality-council/processes/pr-management.md

```diff
@@ -0,0 +1,159 @@
+# Testing Pull Request
+
+
+## How to test a PR manually
+
+You want to take care of a PR and you don't know how to handle it, or you've simply forgotten how. No problem, this github page is for you. 
+
+
+### 1. Taking information from a PR
+
+
+One of the first things to do when starting a PR is to find out as much as possible about it. To do this, you'll find this table at the start of each PR
+
+![img.png](images/readme7.png)
+
+To begin with, we have the **Branch**, which will be useful for preparing our shop and automated tests. So we can already set up a shop accordingly if we don't already have a shop in this branch. 
+
+To do this, go to your terminal and execute the commands below: 
+
+
+ℹ️ For your information, I recommend that you put it in the *Computer/var/www/html* folder.
+
+![img.png](images/readme8.png)
+
+
+We can see that in **violet** we've put the "**Branch "** we're interested in and in **brown** the name of our file, which can be modified as we wish. 
+
+Still with this **Branch**, I advise you to launch the automated Tests. To do this, I suggest you look at the end of this chapter dedicated to it.
```

Comment:
I can't find the chapter dedicated to the launch of automated tests, is it me or it's missing?